test/test.js

```diff
@@ -748,4 +748,16 @@ describe('sanitizeHtml', function() {
         allowedSchemes: sanitizeHtml.defaults.allowedSchemes.concat([ 'tel' ]),
       }), '<q cite=\"http://www.google.com\">HTTP</q><q cite=\"https://www.google.com\">HTTPS</q><q cite=\"mailto://www.google.com\">MAILTO</q><q cite=\"tel://www.google.com\">TEL</q><q cite=\"ftp://www.google.com\">FTP</q><q>DATA</q><q>LDAP</q><q>ACROBAT</q><q>VBSCRIPT</q><q>FILE</q><q>RLOGIN</q><q>WEBCAL</q><q>JAVASCRIPT</q><q>MMS</q>');
   });
+  it('Should encode &, <, > and "', function() {
+    assert.equal(sanitizeHtml('"< & >"'), '&quot;&lt; &amp; &gt;&quot;');
+  });
+  it('Should not encode ampersands on HTML entities when parser.decodeEntities = false', function() {
```

Comment:
Re: _Can't this be used to sneak in entity escapes when typing in a rich text editor, for instance? That doesn't sound like desirable behavior._

I'm not sure about that, but converting things like `&reg; to &amp;reg can't be right. Maybe the test I wrote is wrong, and what it actually needs to do it simply none double encode things?

Reply:
@boutell Thanks for for feedback before. I've now updated the PR and I hope it might be acceptable now.